README.md

```diff
@@ -19,6 +19,10 @@ that incorporates and documents best practices for a performant and scalable des
 security by default for control, logging and evidence generation. It can be  simply deployed by
 customers through a Terraform workflow.
 
+## Disclaimer
+
+To ensure a safe environment we highly recommend you remove all owners roles in the projects used as inputs for *Data Warehouse module*, respecting the principle of [separation of duties](https://cloud.google.com/kms/docs/separation-of-duties).
+
```

Comment:
While this is good, I think it's also important to point users to how they can determine this condition. I think we can help customers check for this condition.  They can use SCC's SHA capability denoted here:
https://cloud.google.com/security-command-center/docs/concepts-vulnerabilities-findings#kms-findings

and have them look for the "owner" finding.  That will then point them to the projects where it was found as well as how to resolve it.

An alternative is to use Cloud Asset Inventory and Search with `gcloud`; see this doc https://cloud.google.com/asset-inventory/docs/searching-iam-policies#examples_query_by_role

Reply:
We added this information. Could you PTAL?